pkg/execute/action.go

```diff
@@ -0,0 +1,88 @@
+package execute
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/kubeshop/botkube/pkg/config"
+)
+
+func (e *DefaultExecutor) runActionCommand(ctx context.Context, args []string, clusterName string) (string, error) {
```

Comment:
Great, that it is defined in a separate file, but can you also define a separate struct, same as the Notifier executor? https://github.com/kubeshop/botkube/blob/main/pkg/execute/notifier.go#L40

In that way you'll be able to pass the actions config there and read it when doing `actions list` 👍